src/SimpleJavaApp/Main.java

```diff
@@ -1,4 +1,18 @@
 package SimpleJavaApp;
 
+import javax.swing.*;
+
 public class Main {
+    public static void main(String[] args) throws ClassNotFoundException, UnsupportedLookAndFeelException, InstantiationException, IllegalAccessException {
```

Comment:
If there are many exceptions and we don't handle the result on each exception, you should simply write
 `  try {
            UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
        } catch (Exception e) {
            e.printStackTrace();
        }`